src/Hosting.Services.Remoting/HostBuilderExtensions.cs

```diff
@@ -64,8 +78,15 @@ public static class HostBuilderExtensions
 				Func<IServiceProvider, TService, IService> createService,
 				FabricTransportRemotingListenerSettings? settings = null)
 					where TService : IServiceFabricService<TContext>
-					where TContext : ServiceContext =>
-						builder.AddServiceListener(p =>
-							new GenericRemotingListenerBuilder<TService>(name, p, createService, settings));
+					where TContext : ServiceContext
+		{
+			if (settings == null)
+			{
+				settings = new FabricTransportRemotingListenerSettings();
+			}
+			settings.ExceptionSerializationTechnique = FabricTransportRemotingListenerSettings.ExceptionSerialization.Default;
+			return builder.AddServiceListener(p =>
```

Comment:
This one doesn't call the other AddRemotingListener extensions, so it will need the explicit setting override anyway.

Reply:
This one calls GenericRemotingListenerBuilder which in turn is child of RemotingListenerBuilder. I think we can remove change from HostBuilderExtensions entirely.